portworx-encryption/assets/create-vols.sh

```diff
@@ -0,0 +1,6 @@
+ssh -o strictHostKeyChecking=no node01 "pxctl volume create testvol"
+
+ssh -o strictHostKeyChecking=no node02 "pxctl volume create local_volume --nodes=LocalNode"
+
+ssh -o strictHostKeyChecking=no node03 "pxctl volume create ha_volume --nodes=LocalNode --repl3"
```

Comment:
--repl3 --> --repl 3

Reply:
this is already fixed. the volume is created when the scenario loads.